src/ralph/data_center/subscribers.py

```diff
@@ -119,19 +127,39 @@ def handle_create_vip_event(data):
     logger.debug('VIP %s created successfully.', vip.name)
 
 
+def migrate_vip_to_cluster(vip, cluster, protocol):
+    msg = (
+        'Trying to update VIP with IP {}, port {} and protocol {}'
+        'failed: %s'.format(vip.ip.address, vip.port, protocol.name)
+    )
+    ethernet = vip.ip.ethernet
+    if not ethernet:
+        logger.error(msg, 'no `Ethernet` object found')
+        return
+    if ethernet.base_object != vip.parent:
+        logger.error(msg, '`Ethernet` base_object differs from `VIP` parent')
+        return
+    if not isinstance(ethernet.base_object, Cluster):
```

Comment:
This does not work (tested):

```
(Pdb++) type(ethernet.base_object)
<class 'ralph.assets.models.base.BaseObject'>
(Pdb++) type(vip.parent)
<class 'ralph.assets.models.base.BaseObject'>

```

```suggestion
    if ethernet.base_object.content_type != cluster_content_type:
```